a10_neutron_lbaas/neutron_ext/db/a10_device_instance.py

```diff
@@ -105,3 +111,10 @@ def get_a10_device_instances(self, context, filters=None, fields=None,
                                     self._make_a10_device_instance_dict, filters=filters,
                                     fields=fields, sorts=sorts, limit=limit,
                                     marker_obj=marker, page_reverse=page_reverse)
+
+    def delete_a10_device_instance(self, context, id):
+        with context.session.begin(subtransactions=True):
+            LOG.debug("A10DeviceInstanceDbMixin:delete_a10_device_instances() id=%s" %
+                      (id))
+            instance = self._get_by_id(context, models.A10DeviceInstance, id)
+            context.session.delete(instance)
```

Comment:
This doesn't delete the instance, it'd need to call instance manager to do that.